deltachat-ios/View/ProviderInfoCell.swift

```diff
@@ -0,0 +1,111 @@
+import UIKit
+
+enum ProviderInfoStatus: Int {
+    case preparation = 2
+    case broken = 3
+ }
+
+class ProviderInfoCell: UITableViewCell {
+
+    private struct ColorSet {
+        let backgroundColor: UIColor
+        let tintColor: UIColor
+    }
+
+    private let fontSize: CGFloat = 13
+
+    private let brokenColorSet: ColorSet = ColorSet(
+        backgroundColor: SystemColor.red.uiColor,
+        tintColor: UIColor.white
+    )
+
+    private let preparationColorSet: ColorSet = ColorSet(
```

Comment:
for preparation-color, can we please use the colors from https://github.com/deltachat/deltachat-android/pull/1219/files#diff-f189d9ec03b8e9f14e2941b3ad306427R62 - the red for broken looks good and it is probably better to use the system-alert-color for that,